Source/Mosa.DeviceSystem/Bitmap.cs

```diff
@@ -6,12 +6,12 @@ public class Bitmap
 	{
 		public static Image CreateImage(byte[] data)
 		{
-			if (data[0] != (byte)'B' || data[1] != (byte)'M')
+			if (data[0] != (byte)'B' && data[1] != (byte)'M')
```

Comment:
if you're checking to see if the data starts with `BM` then this is the wrong operator, the other option is to use `if (!(data[0] == (byte)'B' && data[1] == (byte)'M'))`